Clarify `announce` and `enqueue` is ignored if not supported by targeted media_player (#23171)
home-assistant/home-assistant.github.io: source/_integrations/media_player.markdown

| `media_content_id`     |       no | A media identifier. The format of this is integration dependent. For example, you can provide URLs to Sonos and Cast but only a playlist ID to iTunes.                   |
| `media_content_type`   |       no | A media type. Must be one of `music`, `tvshow`, `video`, `episode`, `channel` or `playlist`. For example, to play music you would set `media_content_type` to `music`. |
| `enqueue`              |      yes | How the new media should interact with the queue. Must be one of `add`, `next`, `play`, `replace`. |

> | `announce`             |      yes | Set to `true` to request the media player to temporarily stop playing media to announce this media and then resume. Most media players don't support `announce` and will not resume playing media once the announcement finishes.

Reviewer: Dangerous to put it with this wording because it requires updating it later. Better make a generic statement.

```suggestion
| `announce`             |      yes | Set to `true` to request the media player to temporarily stop playing media to announce this media and then resume. Requires the media player to support this feature.
```

Author: OK, rephrased to explain what happens when the media player doesn't support `announce` while not quantifying support among integrations 👍
